integration_tests/__tests__/__snapshots__/native-async-mock-test.js.snap

```diff
@@ -1,16 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Snapshots would be problematic on different node versions, resulting in obsolete ones.